src/main/java/core/mate/academy/service/impl/BulldozerProducerImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.mate.academy.service.impl;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducerImpl implements MachineProducer<Bulldozer> {
+    private final List<Bulldozer> bulldozers;
```

Comment:
Consider using access modifiers for the 'bulldozers' field. If it is meant to be immutable from outside of this class, it should be private. Exposing class fields directly is not recommended.